spec/util/miq-password_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'util/miq-password'
 require 'tempfile'
+require 'util/extensions/miq-deep'
```

Comment:
I feel like this should be included in `miq-password.rb` as that's where `deep_clone` is being used.